query/search.js

```diff
@@ -120,25 +120,14 @@ function generateQuery( clean ){
 }
 
 function getQuery(vs) {
-  if (isSingleFieldGeoambiguity(vs) && !hasQueryOrAddress(vs)) {
-    // return `undefined` for now until we exorcise the geodisambiguation demons
-    return;
-  } else {
```

Comment:
For purely readability reasons, it might be nice to leave the else statement with something like

```javascript
} else {
  return undefined; // the old middleware will handle the request if undefined is returned here
}
```